boardesp/webserver.c

```diff
@@ -193,8 +193,35 @@ static void ICACHE_FLASH_ATTR web_rx_cb(void *arg, char *data, uint16_t len) {
       } else {
         ets_strcat(resp, "<br/>esp flash file: user1.bin");
       }
+      
+      ets_strcat(resp,"<br/>Set USB Mode:"
+        "<button onclick=\"location.href='/dcp'\" type='button'>"
```

Comment:
So this navigates away from the main page, can you use XHR to not?

Reply:
Yes, XHR should be fine. Might want to look at something like WebSockets or mqtt long term if we're going to be sending lots of data back and forth.